persistence/sql/persister_jwk.go

```diff
@@ -62,6 +81,30 @@ func (p *Persister) AddKeySet(ctx context.Context, set string, keys *jose.JSONWe
 	})
 }
 
+func (p *Persister) UpdateKey(ctx context.Context, set string, key *jose.JSONWebKey) error {
```

Comment:
UpdateKey does not yet have a test.

I think it's necessary to run this command as a transaction, so that we always DELETE and INSERT the keys (commit), or do neither (rollback).

Reply:
Tests are added to TestManagerKey/TestHelperManagerKeySet. Also see https://github.com/ory/hydra/pull/2625/files#r760934347